src/test/components/__snapshots__/FlameGraph.test.js.snap

```diff
@@ -17,22 +17,7 @@ exports[`FlameGraph EmptyReasons shows reasons when a profile has no samples 1`]
 </div>
 `;
 
-exports[`FlameGraph EmptyReasons shows reasons when samples are out of range 1`] = `
-<div
-  class="EmptyReasons"
->
-  <h3>
-    The 
-    flame graph
-     is empty for “
-    Empty
-    ”
-  </h3>
-  <p>
-    Broaden the selected range to view samples.
-  </p>
-</div>
-`;
+exports[`FlameGraph EmptyReasons shows reasons when samples are out of range 1`] = `null`;
```

Comment:
These snapshot changes indicate that there is a problem in the change. Because it looks like we are not properly indicating the empty thread with `EmptyReasons` properly for flame graph and stack chart.

Reply:
Got it, I'll keep that in mind. It'll help me make sure there aren't any unintended side-effects of the code changes being made.